### PR TITLE
Add create-service-account command

### PR DESCRIPTION
**Pull request by joelhoisko**
## Summary

Adds the new  `create-service-account` command, which creates a  service account called `devops` and adds it to the `developer` role binding

### CODE REVIEW

**Comment by joelhoisko** [comment]
# How to test

1. Reset your local k3d
2. Run the following command to create the serviceaccount
```sh
go run main.go microservice create-service-account 11b6cf47-5d9f-438f-8116-0d9828654657
```
3. Check the service account
```sh
kubectl -n application-11b6cf47-5d9f-438f-8116-0d9828654657 get serviceaccount
```
4. Check the rolebinding
```sh
kubectl -n application-11b6cf47-5d9f-438f-8116-0d9828654657 get rolebindings.rbac.authorization.k8s.io -o wide
```

**Review by freshteapot — APPROVED** [review]
- Great work @joelhoisko.
- Tested locally
- Created a second application to confirm it was happy.
- Some comments, but the tick is there, so you can just read and ignore or make a change.
- Even clicked on the button in my local copy of Studio.

File: cmd/microservice/create-service-account.go [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+		// create the clientset
+		client, err := kubernetes.NewForConfig(config)
+		if err != nil {
+			panic(err.Error())
+		}
+
+		k8sRepo := platform.NewK8sRepo(client, config)
+
+		createAll := viper.GetBool("all")
```
[the remark this answers is not in the document]
Comment (joelhoisko) [reply cmd/microservice/create-service-account.go:51]: I changed the output to show the results more nicely:
```sh
go run main.go microservice create-service-account --all
{"level":"info","msg":"Adding a devops service account for all applications","time":"2021-11-11T10:05:24+01:00"}
{"applicationID":"11b6cf47-5d9f-438f-8116-0d9828654657","customerID":"453e04a7-4f9d-42f2-b36c-d51fa2c83fa3","level":"info","method":"createServiceAccount","msg":"Added a service account for application 11b6cf47-5d9f-438f-8116-0d9828654657","serviceAccount":"devops","time":"2021-11-11T10:05:24+01:00"}
{"level":"info","msg":"Added 1 service accounts","time":"2021-11-11T10:05:24+01:00"}
{"level":"info","msg":"Finished!","time":"2021-11-11T10:05:24+01:00"}
```

Running this command again:
```sh
go run main.go microservice create-service-account --all
{"level":"info","msg":"Adding a devops service account for all applications","time":"2021-11-11T10:07:46+01:00"}
{"level":"info","msg":"Application '11b6cf47-5d9f-438f-8116-0d9828654657' already had the service account or rolebinding, skipping","time":"2021-11-11T10:07:46+01:00"}
{"level":"info","msg":"Added 0 service accounts","time":"2021-11-11T10:07:46+01:00"}
{"level":"info","msg":"Finished!","time":"2021-11-11T10:07:46+01:00"}
```

File: cmd/microservice/create-service-account.go [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+		}
+
+		customerID := k8sNamespace.Annotations["dolittle.io/tenant-id"]
+		customerName := k8sNamespace.Labels["tenant"]
+		applicationName := k8sNamespace.Labels["application"]
+		addServiceAccount(logContext, k8sRepo, customerID, customerName, applicationID, applicationName)
+		if err != nil {
+			panic(err.Error())
+		}
+		logContext.Infof("Created 'devops' serviceaccount for application %s", applicationID)
```
[the remark this answers is not in the document]
Comment (joelhoisko) [reply cmd/microservice/create-service-account.go]: I changed the output to show the result more nicely (for both `--all` and the single approach), here's for creating a single svc acc:
```sh
go run main.go microservice create-service-account 11b6cf47-5d9f-438f-8116-0d9828654657
{"applicationID":"11b6cf47-5d9f-438f-8116-0d9828654657","customerID":"453e04a7-4f9d-42f2-b36c-d51fa2c83fa3","level":"info","method":"createServiceAccount","msg":"Added a service account for application 11b6cf47-5d9f-438f-8116-0d9828654657","serviceAccount":"devops","time":"2021-11-11T10:10:23+01:00"}
{"level":"info","msg":"Finished!","time":"2021-11-11T10:10:23+01:00"}
```

Running it again:
```sh
go run main.go microservice create-service-account 11b6cf47-5d9f-438f-8116-0d9828654657
{"level":"info","msg":"Application '11b6cf47-5d9f-438f-8116-0d9828654657' already had the service account or rolebinding, skipping","time":"2021-11-11T10:10:47+01:00"}
{"level":"info","msg":"Finished!","time":"2021-11-11T10:10:47+01:00"}
```

**Review by freshteapot — APPROVED** [review]
:)

**Pull request merged**